lib/spack/spack/cmd/install.py

```diff
@@ -48,6 +48,9 @@ def setup_parser(subparser):
         '-n', '--no-checksum', action='store_true', dest='no_checksum',
         help="Do not check packages against checksum")
     subparser.add_argument(
+        '-p', '--parallel', action='store_true', dest='build_parallel',
```

Comment:
That's a good point indeed. As the make commands aren't related the `-j` is counted individually. I could add an `-l` to the install command in general. And set it to some reasonable default whenever parallel builds are requested.